src/writing-policies/rego/gatekeeper/04-distribute.md

```diff
@@ -0,0 +1,88 @@
+# Distribute
+
+Policies have to be annotated for them to be pushed, and eventually
+executed by the Kubewarden `policy-server` in a Kubernetes cluster.
+
+Annotating and distributing our gatekeeper policy is very similar to
+distributing an Open Policy Agent one. Let's go through it.
+
+## Annotating the policy
+
+We are going to write a `metadata.yaml` file in our policy directory
+with contents:
+
+```yaml
+rules:
+- apiGroups: [""]
+  apiVersions: ["*"]
+  resources: ["*"]
+  operations: ["CREATE"]
+mutating: false
+contextAware: false
+executionMode: gatekeeper
+annotations:
+  io.kubewarden.policy.title: no-default-namespace
+  io.kubewarden.policy.description: This policy will reject any resource created inside the default namespace
+  io.kubewarden.policy.author: The Kubewarden Authors
+  io.kubewarden.policy.url: https://github.com/kubewarden/some-policy
+  io.kubewarden.policy.source: https://github.com/kubewarden/some-policy
+  io.kubewarden.policy.license: Apache-2.0
+  io.kubewarden.policy.usage: |
+      This policy is just an example.
+
+      You can write interesting descriptions about the policy here.
+```
+
+As you can see, everything is the same as the Open Policy Agent
+version metadata, except for the `executionMode: gatekeeper` bit.
+
+Let's go ahead and annotate the policy:
+
+```console
+gatekeeper-policy » kwctl annotate policy.wasm --metadata-path metadata.yaml --output-path annotated-policy.wasm
```

Comment:
I think it would be better to just replace the shell prompt with a simple `$`, it keeps everything more concise. That applies to other prompts around.

That's a matter of personal taste :sweat_smile: